dev_guide/daemonsets.adoc

```diff
@@ -1,5 +1,5 @@
 [[dev-guide-daemonsets]]
-= Using Daemonsets
+= Using DaemonSets
```

Comment:
Please update the title in the _topic_map.yaml file.

I see a couple of other uses of "Daemonsets" in the collection. I'd consider fixing them on this PR or a follow-up PR.

Reply:
@kalexand-rh Updated instances of Daemonset and daemonset to DaemonSet.